dockers/base-cuda/Dockerfile

```diff
@@ -119,7 +119,7 @@ RUN \
 RUN \
     # install DeepSpeed
     # TODO(@SeanNaren): 0.3.15 is broken - skipping to unblock
```

Comment:
```suggestion
    # TODO(@SeanNaren): >=0.3.15 is broken - skipping to unblock
```

Reply:
I will try it but I doubt that it's a deepspeed problem. The comment is probably referring to us having a bug with the new deepspeed version, not the other way around.

Reply:
yes - just proposed this to keep the comment up to date

```suggestion
    # TODO(@SeanNaren): CI failing with `>=0.3.15` - skipping to unblock
```